soccer/gameplay/skills/capture.py

```diff
@@ -99,7 +99,8 @@ def on_enter_course_approach(self):
 
     def execute_course_approach(self):
         # don't hit the ball on accident
-        self.robot.set_avoid_ball_radius(Capture.CourseApproachAvoidBall)
+        #self.robot.set_avoid_ball_radius(Capture.CourseApproachAvoidBall)
+        self.robot.disable_avoid_ball()
```

Comment:
Ahh, I gotcha - I think I was reading "pos" as "robot's current position" on that line - my mistake.

It looks like `find_intercept()` will sometimes return a point less than the course approach dist since it samples increments of 5cm from the ball until it finds a good point, while the course approach dist is 10cm.

I think what you've got now is good :+1: